week03_model_free/homework.ipynb

```diff
@@ -766,6 +766,49 @@
    },
    "outputs": [],
    "source": []
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "%%writefile ../run_display_servers.sh\n",
+    "# OPEN PORT 6080!\n",
+    "apt-get update\n",
+    "apt-get install -y python-tk\n",
+    "apt-get install -y x11vnc\n",
+    "x11vnc -R stop\n",
+    "x11vnc -bg -forever -display :1 -rfbport 5900\n",
+    "ls noVNC > /dev/null && rm -r noVNC\n",
+    "git clone https://github.com/novnc/noVNC.git\n",
+    "cp noVNC/vnc.html noVNC/index.html\n",
+    "# Do not work if it run in Jupyter\n",
+    "#head -n 166 noVNC/utils/launch.sh > noVNC/utils/bglaunch.sh\n",
+    "#chmod ago+x noVNC/utils/bglaunch.sh\n",
+    "#noVNC/utils/bglaunch.sh --vnc localhost:5900 --listen 6080\n",
+    "# Instead run in a terminal window ...\n",
+    "# noVNC/utils/launch.sh --vnc localhost:5900 --listen 6080\n",
+    "# ... and go to http://servername:6080 page"
```

Comment:
I don't understand these instructions at all, sorry.

Reply:
I want to show, that hacks of running from Jupyter was tried and they are not good enough.